src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    private static final Random random = new Random();
+    private final Color[] colors = Color.values();
+
+    public static String getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index].name().toLowerCase();
+    }
```

Comment:
Don't use verbs for class/interface names. In this case, the class name 'ColorSupplier' is appropriate as it follows the noun convention for class names.